scaletest/templates/scaletest-runner/scripts/lib.sh

```diff
@@ -271,10 +271,12 @@ fetch_coder_full() {
 		exit 1
 	fi
 	log "Fetching full Coder binary from ${pod}"
+	# We need --retries due to https://github.com/kubernetes/kubernetes/issues/60140 :(
 	maybedryrun "${DRY_RUN}" kubectl \
 		--namespace "${namespace}" \
 		cp \
 		--container coder \
+		--retries 3 \
```

Comment:
Only 3? 😅 Maybe we'll go a bit nuttier? 😉

```suggestion
		--retries 10 \
```

Reply:
lol the linked github issue has folks setting 999 retries so 10 is relatively tame by comparison